scripts/export-git-env-vars.sh

```diff
@@ -3,15 +3,21 @@ set -euo pipefail
 set -x
 
 
+# If enabled, keep my tags intact, it won't run git fetch --prune
+KEEP_MY_TAGS_INTACT=${KEEP_MY_TAGS_INTACT:-1}
+
 # In case of running this script on detached head, script has difficulties in finding out
 # what is the current 
 echo "Exporting Git Variables: "
 
-git fetch
-
 function find_most_recent_numeric_tag() {
-    # We use the --prune flag because we've had problems with buildkite agents getting conflicting results here
-    git fetch --tags --prune --prune-tags --force
+
+    if [[ "$KEEP_MY_TAGS_INTACT" != 1 ]]; then
+        # We use the --prune flag because we've had problems with buildkite agents getting conflicting results here
+        git fetch --tags --prune --prune-tags --force
+    else
+        git fetch --tags --force
```

Comment:
It was scary on my side to see many branches being removed. Is it necessary for a local development?

Reply:
Do you want me to remove fetching if flag for not touching tags is enabled?

Reply:
> Do you want me to remove fetching if flag for not touching tags is enabled?

Yes.

Reply:
It would be nice as it adds useless logs to the stdout for the user.